src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java

```diff
@@ -120,6 +125,10 @@
     public void start(Promise<Void> prom) {
         log.info("Starting verticle");
 
+        Vertx vertx = Vertx.vertx(new VertxOptions().setMetricsOptions(
+                new MicrometerMetricsOptions()
+                    .setPrometheusOptions(new VertxPrometheusOptions().setEnabled(true))
```

Comment:
Do we want to enable/disable here too?

Reply:
Yeah I suppose. Also I forgot to add it to `src/dist/conf/config.yml`

Reply:
Actually, right now we need the Vertx instance before we retrieve the config, so we don't have the `vertx-metrics:enabled` information when we initialize the vertx instance.

Reply:
Yeah, creating a new vertx is also not really ideal either since we already have an instance here. Definitely has the possibility of being troublesome.